README.md

```diff
@@ -77,8 +77,45 @@ Once an image has been built, [push-to-registry](https://github.com/redhat-actio
     <td><a href="https://github.com/openshift/source-to-image/blob/master/docs/cli.md#log-levels">Log level</a> when running S2I. Can be 0 (least verbose) to 5 (most verbose).</td>
   </tr>
 
+  <tr>
+    <td>env_vars</td>
+    <td>No</td>
+    <td>-</td>
+    <td>List of Environment variable to be passed to the builder (eg. <code>key=value</code>, <code>mysecret=${{ secrets.value }}</code> ).</td>
+  </tr>
+
 </table>
 
+### Multi-line envoirnment variables
```

Comment:
I don't think we have to document here how to write environment variables. It follows the same rules as the regular `env` section, right?

Reply:
Yes, it does.